sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -721,6 +721,34 @@ foreach (MetricsQueryResult value in metricsQueryResults.Values)
 }
 ```
 
+The `MetricsQueryResourcesOptions`-typed argument also has a `StartTime` and `EndTime` property to allow for querying a specific time range. If only the starttime is set, the endtime default becomes the current time. When the endtime is specified, the starttime is necessary as well. The following example demonstrates the use of these properties:
```

Comment:
The properties are named Start and End currently, though I think it would be better to use StartTime and EndTime. Start/End made more sense on QueryTimeRange because that type already has "time" in the name.